dorpsgek_github/github.py

```diff
@@ -0,0 +1,182 @@
+import datetime
+
+import aiohttp
+import jwt
+import time
+
+from gidgethub import (
+    routing,
+    sansio,
+)
+from gidgethub.aiohttp import GitHubAPI
+from gidgethub.sansio import accept_format
+
+from dorpsgek_github.config import (
+    GITHUB_APP_ID,
+    GITHUB_APP_PRIVATE_KEY,
+    GITHUB_APP_SECRET,
+)
+
+
+router = routing.Router()
+_github_tokens = {}
+_github_installations = {}
+
+
+# HACK - Monkeypatch a fix that is already upstream
```

Comment:
link to commit that fixes this? Or an expected release number? Similar for the other "HACK"s